package-system/astc-encoder/install_linux.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+BIN_PATH=$TARGET_INSTALL_ROOT/bin
+INCLUDE_PATH=$TARGET_INSTALL_ROOT/include
+
+mkdir -p $INCLUDE_PATH
+mkdir -p $BIN_PATH
+
+# copy LICENSE.txt and header file
+cp -f temp/src/LICENSE.txt $TARGET_INSTALL_ROOT/
+cp -f temp/src/Source/astcenc.h $INCLUDE_PATH/
+
+BUILD_PATH=temp/build/Source
+
+# copy static lib and executable
+cp -f $BUILD_PATH/astcenc-sse2 $BIN_PATH/
+cp -f $BUILD_PATH/libastcenc-sse2-static.a $BIN_PATH/
```

Comment:
It may be useful here to fail the script if the cp fails.
In unix bash its prety easy
```
(command) || exit $?
```

like
```
cp -f temp/src/LICENSE.txt $TARGET_INSTALL_ROOT/ || exit $?
```
will cause it to either succeed the copy OR exit with the exit code of cp.

Note that this file does not go into the package, so changing it would not require a re-upload.